src/ssl.c

```diff
@@ -14092,17 +14115,12 @@ int wolfSSL_GetSessionFromCache(WOLFSSL* ssl, WOLFSSL_SESSION* output)
             preallocNonce, &preallocNonceLen, &preallocNonceUsed);
 #else
         error = wolfSSL_DupSession(sess, output, 1);
-#endif /* WOLFSSL_TSL */
-
+#endif /* HAVE_SESSION_TICKET && WOLFSSL_TLS13 */
 #ifdef HAVE_EX_DATA
         output->ownExData = 0; /* Session cache owns external data */
 #endif
+        TlsSessionCacheUnlockRow(row);
```

Comment:
It looks like the row can be unlocked twice.

Reply:
my bad, I wrongly refactored an if. Thanks.
Added a couple of fixes for the clang static analyzer as well.